### ISSUE TITLE
incorrect escaping if delimiter character different from /

### ISSUE THREAD

**Issue by Konfekt** [osyo-manga/vim-over#39]
For example `:%substitute:\va:b:cgI` will incorrectly highlight `a:b`.

**Comment by osyo-manga** [comment]
Thanks for issues.
However, it did not reproduce. :(
![](http://i.gyazo.com/eb6954dc1fb30c1864576e4e6e5c9467.png)

**Comment by Konfekt** [comment]
Sorry, the mistake is rather that

`:%substitute:\va::b::cgI` will incorrectly highlight `a:` with `b:`, whereas `Vim` throws an error.

**Comment by osyo-manga** [comment]
Please use separator `/`.
`:%substitute/\va:/b:/cgI`

**Comment by Konfekt** [comment]
yeah that's the default. But Vim accepts `:` as well. Just to point out that inconsistency to `Vim`.

**Comment by osyo-manga** [comment]
Sorry but there is no way to do that because of Vim's limitation. Please just use `/`.

**Comment by Konfekt** [comment]
Ok. Just wanted to file it.
